.travis.yml

```diff
@@ -9,7 +9,10 @@ go_import_path: github.com/containerd/console
 install:
   - go get -d
   - GOOS=windows go get -d
+  - GOOS=solaris go get -d
 
 script:
   - go test -race
   - GOOS=windows go test
+  - GOOS=solaris go build
+  - GOOS=solaris go test -c
```

Comment:
hmm, dont we want to run go test for solaris here ? `go test -c` just compile the tests without running.